config/providers.yml

```diff
@@ -17,5 +17,12 @@
       {display_name: us-east-a2, internal_name: leaseweb-wdc02, ui_name: "Virginia, US", visible: true},
       {display_name: us-south-a13, internal_name: leaseweb-dal13,  ui_name: "Dallas, US", visible: false},
     ]
+  },
+  {
+    provider_display_name: Latitude,
+    provider_internal_name: latitude,
+    locations: [
+      {display_name: eu-central-l1, internal_name: latitude-fra, ui_name: "Frankfurt, Germany", visible: false}
```

Comment:
Currently, Hetzner regions are named with countries, while others use both city and country names. I recall Umur initiating a discussion on this topic earlier. I can't remember the outcome, but I believe we should adhere to one naming convention.

Reply:
There is already an active discussion in slack about this :)